recipes/mpir/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "2.7.2" %}
+
+package:
+  name: mpir
+  version: {{ version }}
+
+source:
+  fn: mpir-{{ version }}.tar.bz2
+  url: http://mpir.org/mpir-{{ version }}.tar.bz2
+  md5: b31aa1ebc01b5fedf4ac43277c03feec
+
+build:
+  number: 0
```

Comment:
skip: True  # [win and py<35]

might be a little better.  I would not bother adding vc9 and vc10 features, since we are not building on those compilers.